hw_9_1.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<p>Since you are using FFT here, please use the appropriate GitHub repository and open a pull request there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>